mobile/voice_journal_app/lib/calander.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart'; // library for date calculations, formatting, locales, etc...
+import 'day_details.dart'; // details page
+
+void main() {
+  runApp(CalendarApp());
+}
+
+class CalendarApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Calendar',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: CalendarPage(),
+    );
+  }
+}
+
+class CalendarPage extends StatefulWidget {
+  @override
+  _CalendarPageState createState() => _CalendarPageState();
+}
+
+class _CalendarPageState extends State<CalendarPage> {
+  DateTime _selectedDate = DateTime.now();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color.fromRGBO(252, 252, 252, 1),
```

Comment:
Let's use the colors defined in [theme.dart](https://github.com/LaurierCS/POD1/blob/main/mobile/voice_journal_app/lib/theme.dart), perhaps `lightGray`? This will be good for consistency across the app.

Reply:
Let's do the same for the other colors on this page. Feel free to add new colors to the `theme.dart` file if you need to.